framework/src/executioners/Transient.C

```diff
@@ -536,7 +536,8 @@ Transient::computeConstrainedDT()
   }
 
   // Adjust to a target time if set
```

Comment:
Please improve this to explain this rather complicated if statement.

Reply:
Changed to: `// If a target time is set and the current dt would exceed it, limit dt to match the target`